pinery-miso/src/main/java/ca/on/oicr/pinery/lims/miso/converters/SampleTypeConverter.java

```diff
@@ -0,0 +1,132 @@
+package ca.on.oicr.pinery.lims.miso.converters;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SampleTypeConverter {
+
+  private static final Logger log = LoggerFactory.getLogger(SampleTypeConverter.class);
+
+  private static final String MISO_TYPE_LIBRARY = "Library";
+  private static final String MISO_TYPE_DILUTION = "Dilution";
+
+  private static final String PLATFORM_ILLUMINA = "ILLUMINA";
+
+  private static final String LIBRARY_TYPE_MRNA = "mRNA Seq";
+  private static final String LIBRARY_TYPE_PAIRED_END = "Paired End";
+  private static final String LIBRARY_TYPE_SMALL_RNA = "Small RNA";
+  private static final String LIBRARY_TYPE_SINGLE_END = "Single End";
+  private static final String LIBRARY_TYPE_WHOLE_TRANSCRIPTOME = "Whole Transcriptome";
+  private static final String LIBRARY_TYPE_MATE_PAIR = "Mate Pair";
+  private static final String LIBRARY_TYPE_TOTAL_RNA = "Total RNA";
+
+  private static enum IlluminaSampleType {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this redundant "static" qualifier. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2786)